File: sort/bubble_sort.c
void bubble_sort(int arr[], int n){
    int i, j, key, tmp;
    for (i=n-1; i>0; i--) {
        for (j=0; j<i; j++) {
            if (arr[j] > arr[j + 1]) {
                tmp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = tmp;
            }
        }
    }
}

File: sort/bubble_sort.py

def bubble_sort(a):
    for i in range(len(a) - 1, 1, -1):
        for j in range(i):
            if a[j] > a[j + 1]:
                a[j], a[j + 1] = a[j + 1], a[j]
